### PR TITLE
[T4A7][T11-B2]Sun Mingyang

### PR DESCRIPTION
**Pull request by nightism**

### CODE REVIEW

File: src/seedu/addressbook/data/person/Name.java [hunk 1]
```diff
@@ -43,6 +43,18 @@ public static boolean isValidName(String test) {
...
         return Arrays.asList(fullName.split("\\s+"));
     }
+    
+    /**
+     * Returns true of the other name is very similar to this name.
+     * Two names are considered similar if ...
+     */
+     public boolean isSimilar(Name other) {
+    	 if(other == null) {
+    		 return false;
```
Comment (pyokagan) [inline src/seedu/addressbook/data/person/Name.java]: @nightism Mixing tabs and spaces again.

Comment (nightism) [inline src/seedu/addressbook/data/person/Name.java]: Thanks for reminding.
have newly committed with improved indentation.

Comment (pyokagan) [inline src/seedu/addressbook/data/person/Name.java]: @nightism lmao you are supposed to use spaces for indentation, not tabs XD

**Comment by xdshang** [comment]
Accepting this PR without a review because another of your PRs for this tutorial was reviewed in detail.

**Pull request closed without merging**